invenio_access/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2015, 2017 CERN.
```

Comment:
from the history, this file has been modified in 2016, thus the copyright should be inclusive: `2015-2017`